src/Sulu/Bundle/TestBundle/Behat/DefaultContext.php

```diff
@@ -113,6 +113,16 @@ public function iFillTheSelector($selector, $value)
     }
 
     /**
+     * @Given I leave the selector :selector
+     */
+    public function iLeaveTheSelector($selector)
+    {
+        $this->waitForSelector($selector);
+        $this->getSession()->evaluateScript("$('$selector').trigger('change')");
```

Comment:
Is this necessary? If you only leave an element, it only triggers `focusout`, not `change`. `change` should be triggered by the component itself, if it is required, shouldn't it?